docs/getting_started/run_schedule.md

```diff
@@ -70,7 +79,12 @@ environment, you can import and execute it directly:
 
 from workflows.example import wf
 
-execution = remote.execute(wf, inputs={"name": "Kermit"})
+execution = remote.execute(
+    wf,
+    inputs={"name": "Kermit"},
+    image_config=ImageConfig.auto(img_name="ghcr.io/flyteorg/flytekit:py3.11-1.9.1"),
+    version="v1",
```

Comment:
we don't need this. you saw that error because you didn't register the workflow. could you add a note above to tell people to register a workflow before running execute. `pyflyte register wf.py`